src/store/conversationsStore.js

```diff
@@ -439,6 +440,20 @@ const actions = {
 		commit('addConversation', conversation)
 	},
 
+	async markConversationUnread({ dispatch, getters }, { token }) {
+		const conversation = Object.assign({}, getters.conversations[token])
+		if (!conversation) {
+			return
+		}
```

Comment:
**issue (blocking):** `!conversation` is always falsy, because `conversation` is always an object (`thruly`)

This `if` never works and `conversation` is not used outside this `if`.

I assume, the goal was to check if `getters.conversations[token]` exists? Then we should not clone the object. But in this I'm not sure, that we need to return. 

Is it possible that this action will be called to not existed conversation?

Reply:
I copied this from "read" method, so now it should be checked at both places

Reply:
So, getter always returns an object or undefined, and I don't see any reason to use Object.assign here,
 and in another method (but we must look at Git blame, to understand when and why it was initially added)